packages/react/src/components/CTA/__stories__/CTA.stories.js

```diff
@@ -47,8 +47,10 @@ storiesOf('Components|CTA', module)
           heading: copy[0],
           card: {
             type: type,
-            href: urlBy[type],
-            title: copy[1],
+            heading: copy[1],
+            cta: {
+              href: urlBy[type],
+            },
```

Comment:
@photodow Don't think this needs to be part of this review. Will make note to address for that component.

Reply:
I think I understand why you're saying that, but if you would allow me to push back just a little here. We made updates to the card, and the CTA is supposed to be a 1:1 of the CTA backing it. We also made updates to reflect the changes in the `featureCard` here, and now it's inconsistent when the `card` model above. I think we might live if we do this in a follow up PR, but I do feel like this is relevant still.

Reply:
so in the CTA story
```
case 'card':
        type = select('type', types, types[0]);
        cta = {
          href: urlBy[type],
          heading: copy[0],
        };
        break;
```

in the `CTA.js` itself, we still need to define the icon for the card
```
case 'card':
      return (
        <Card
          customClassName={`${prefix}--card--CTA`}
          cta={{
            href: cta.href,
            icon: {
              src: _iconSelector(type),
            },
          }}
          heading={cta.title}
          type="link"
          ...
        />
      );
```
not sure if we can do a 1:1  as we define the icon used for the `card` within the `CTA` component itself and not the props passed in from the story

Reply:
@annawen1 not sure I understand. Can we sync up first thing in the morning?